packages/core/src/env-set/append-dot-env.ts

```diff
@@ -0,0 +1,7 @@
+import { appendFileSync } from 'fs';
+
+const appendDotEnv = (key: string, value: string) => {
```

Comment:
In some runtime environments like Docker, `ENV` is not passed by `.env` file. Or, the user may set `ENV` directly by OS's `export`, can we detect that and throw an error while tring to append dot env?

Reply:
per discussion add `--silent` option to directly throw error when required env not found, LOG-2254